upgrade-guide.md

```diff
@@ -10,6 +10,8 @@ XXXX
 
 - **VertexIdentifier**: `MeshComponentVertex` has been renamed in `ComponentMeshVertex` and its methods `mesh_component_vertices` in `component_mesh_vertices`.
 
+- **PointsSort**: `morton_sort` has been renamed in `morton_mapping`.
```

Comment:
@MelchiorSchuh what is the (American) English idiom for "is renamed xxx"? I found there is no "in", the correct answer is: A has been renamed B.